src/app/pages/cases/details/DetailsPage.tsx

```diff
@@ -59,6 +59,10 @@ const DetailsPage: React.FC<RouteComponentProps<Props>> = ({ id }) => {
             <ButtonLink to={ to("/cases/:id/correspondence", { id: id })}>
               <ButtonTertiary variant="tertiary">Correspondentie</ButtonTertiary>
             </ButtonLink>
+            &nbsp;
```

Comment:
Can we solve this with CSS (horizontal padding / margin)?

Reply:
Yes, fixed it